awx/main/utils/external_logging.py

```diff
@@ -57,7 +57,11 @@ def construct_rsyslog_conf_template(settings=settings):
             'action.resumeRetryCount="-1"',
             'template="awx"',
             'errorfile="/var/log/tower/rsyslog.err"',
-            f'action.resumeInterval="{timeout}"'
+            f'action.resumeInterval="{timeout}"',
+            'queue.filename="awx-external-log"',
+            'queue.size="100000"',
+            'queue.maxdiskspace="1g"',,
```

Comment:
> Disk assisted queues are special in that they do not have any size limit. The enqueue an unlimited amount of elements. To prevent running out of space, disk and disk-assisted queues can be size-limited via the “$<object>QueueMaxDiskSpace” configuration parameter. If it is not set, the limit is only available free space (and reaching this limit is currently not very gracefully handled, so avoid running into it!).

Reply:
Do we want the user to be able to set/configure this? Once customers get wind of this I could see them saying "Yeah, we are required to log all the things all the time so we need to set this to 1TB. We don't care that it might DDOS our Splunk when it comes back up"

Reply:
I'm not against having a AWX_LOGGING_MAX_DISK_QUEUE_SIZE parameter, but I don't think it's needed for the MVP.

Reply:
Yea, agreed.  I've also updated this to use disk-backed queues, because in my testing, it's much more resistant to risk of OOM.

Reply:
Given that prior versions don't queue or retry *at all*, my inclination is that I agree with Bill - let's release something reasonable here, and if people ask for tunables, we can always  expose them through more AWX settings.